modules/ui/fields/input.js

```diff
@@ -114,8 +116,12 @@ export function uiFieldText(field, context) {
                     var raw_vals = input.node().value || '0';
                     var vals = raw_vals.split(';');
                     vals = vals.map(function(v) {
-                        var num = parseFloat(v.trim(), 10);
-                        return isFinite(num) ? clamped(num + d) : v.trim();
+                        v = v.trim();
+                        var num = parseLocaleFloat(v);
+                        if (!isFinite(num)) return v;
+                        num = parseFloat(num, 10);
+                        if (!isFinite(num)) return v;
+                        return formatFloat(clamped(num + d));
```

Comment:
Can you example what the `num = parseFloat(num)` is supposed to do here? Isn't `num` already numeric after parsing of `v`? Can't this and the next line be just dropped?